lms/djangoapps/certificates/tests/test_api.py

```diff
@@ -0,0 +1,259 @@
+"""Tests for the certificates Python API. """
```

Comment:
Note to reviewers: This looks like I added more code than I actually did.  I moved `tests_api.py` to `test_api.py`, then added some new tests at the bottom.